client_options.go

```diff
@@ -50,6 +50,22 @@ func WithCustomLimiter(limiter RateLimiter) ClientOptionFunc {
 	}
 }
 
+// WithCustomLogger can be used to configure a custom retryablehttp logger
+func WithCustomLogger(logger retryablehttp.Logger) ClientOptionFunc {
+	return func(c *Client) error {
+		c.client.Logger = logger
+		return nil
+	}
+}
+
+// WithCustomLogger can be used to configure a custom retryablehttp leveled logger
+func WithCustomLeveledLogger(leveledLogger retryablehttp.LeveledLogger) ClientOptionFunc {
+	return func(c *Client) error {
+		c.client.Logger = leveledLogger
+		return nil
+	}
+}
```

Comment:
Can you switch them to keep the order alphabetical? Thanks!

Reply:
Yes sure. Done!
However, WithCustomLeveledLogger < WithCustomLimiter, should we bring it before the Limiter too?
Or considering CustomLeveledLogger and CustomLogger are closely related it's fine just switching them?

Reply:
No, good point... Please make them all alphabetically ordered. Thanks, I missed that one 👍🏻

Reply:
Done!